### DISCUSSION TITLE
Can't see what I drew

### DISCUSSION THREAD

**Discussion by SeowXiangHer0399** [FreeCAD/FreeCAD#17963] [Discussions that will turn in to issues but need refining]
![image](https://github.com/user-attachments/assets/52d30d30-33c3-4d91-b74a-5c9215121bca)
Why export file to STEP with colors （*.step *.stp) draw at freecad 1.0.0RC4 cannot see clear?

**Comment by luzpaz** [comment]
Can you post on the forum or discord ? You'll get better traction there. You're also going to need a lot more info. Full Version info, example file etc...
